helm/rpm-builder/templates/flotta-rpm-build-task.yml

```diff
@@ -0,0 +1,53 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  namespace: {{ .Values.targetNamespace }}  
+  name: flotta-build-task
+spec:
+  workspaces:
+    - name: workspace
+  params:
+    - name: source-dir
+      description: Directory Contaning source File
+      type: string
+  results:
+    - name: build-commit
+      description: Commit ID of the Build
+  steps:
+    - name: build-image-playbook
+      image: {{ .Values.buildTask.image.name }}:{{ .Values.buildTask.image.tag }}
+      workingDir: /workspace/workspace
+      args:
+        - |
+          ansible-galaxy collection install -r tools/ansible/collections/requirements.yaml
+          ansible-playbook \
+            -e source_dir=$(params.source-dir) \
+            -e build_arch=`uname -i` \
```

Comment:
Continuing the comment about building yggdrasil for only one arch. Since I don't see the pipeline running on ARM, this will always build x86_64, no?

Reply:
Correct, so far only x86_64